tests/decomposition/test_decomposition_graph.py

```diff
@@ -77,7 +128,7 @@ def custom_hadamard_2(wires):
 
         graph = DecompositionGraph(
             operations=[qml.Hadamard(0)],
-            gate_set={"RX", "RY", "RZ"},
+            gate_set={"RX": 1.0, "RY": 1.0, "RZ": 1.0},
```

Comment:
Not necessary. This test case tests the `_get_decompositions` method, which does not involve the gate set at all.